src/cli.rs

```diff
@@ -14,6 +14,9 @@ pub enum Command {
     #[options(help = "dump font information")]
     Dump(DumpOpts),
 
+    #[options(help = "parse the supplied font, reporting any failures")]
+    Parse(ParseOpts),
```

Comment:
Would `validate` be a better name? `parse` seems to imply that it would produce some output. 🤔

Reply:
I like it! That does seem like a better name.

Reply:
Rename done.